### PR TITLE
New blog post on presenting at Latencyconf

### PR DESCRIPTION
**Pull request by mechanicalpete**

### CODE REVIEW

File: _posts/2019-11-19-s2o-how-it-was.markdown [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+## On the Day
+
+![Pete freaking face with banner]({{ site.url }}/img/inception-pipelines/s2o-how-it-was/banner-freaking.jpg)
+
+Last year at [Latency 2018](https://www.latencyconf.io/#/pastconferences), I presented a lightening talk called "Implementing AWS X-Ray in your lambdas in less than 5 minutes, with cheat sheets!" to approximately 200 people. The talk went quite well, and I wasn't nervous beforehand, partly because I had a cold and couldn't see the crowd. But this year it felt soooo much different!! Check out the crowd below!
```
Comment (jgunnink) [inline _posts/2019-11-19-s2o-how-it-was.markdown]: speeling of lightning

File: _posts/2019-11-19-s2o-how-it-was.markdown [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+After getting my story straight, I spent many hours talking to myself reading out the story and refining it. I can only image what people around me thought!! Still, it was invaluable as from here I was able to distil it all down to a bunch of index cards with the points I wanted to cover. It was also invaluable in helping me in timing parts of the talk. Remember, it is a live-coding presentation so knowing where I need to add 'filler' and where to wrap things up was key. I couldn't face the horror of standing in front of 300 people, crickets chirping, having nothing to say while I waited for the code to catch-up. A bonus was that continually presenting it helped immensely in ironing out all the sharp bits that kept poking me; I have confidence the code would deploy and run smoothly without errors and in the allocated time.
```
Comment (jgunnink) [inline _posts/2019-11-19-s2o-how-it-was.markdown]: distill

File: _posts/2019-11-19-s2o-how-it-was.markdown [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+![The crowd]({{ site.url }}/img/inception-pipelines/s2o-how-it-was/banner-crowding.jpg)
+
+15 minutes beforehand, I had run my run sheet up, everything was primed and ready and I was just counting down. Tick tock, tick tock, tick tock. Finally, I was ready to go! Up I jumped, plugged in my laptop and it was show time. A special call out, and thank-you, to Min for the fantastic introduction.
+
+I feel I got off to a pretty good start and covered almost all the points that I wanted to cover in my introduction. The first hurdle that I needed to cross was getting my code seeded and building in AWS. I was a little ahead of schedule and needed a little bit ad-libbing, but everything was going well. The next hurdle was running the game and again everything operated smoothly. The final hurdle was my downfall. This hurdle was where I ran a load-testing tool that was supposed to overload the game. Except it didn't. Turns out I just didn't have enough bandwidth to kill it. What I needed, from past runs, was about 6-7000 hits within 3 minutes; and I managed to just get to 3000.
```
Comment (jgunnink) [inline _posts/2019-11-19-s2o-how-it-was.markdown]: > What I needed, from past runs, was about 6-7000

Don't need all the commas

File: _posts/2019-11-19-s2o-how-it-was.markdown (hunk 1 above)
Comment (mechanicalpete) [reply _posts/2019-11-19-s2o-how-it-was.markdown]: Good pickup, fixed

File: _posts/2019-11-19-s2o-how-it-was.markdown (hunk 2 above)
Comment (mechanicalpete) [reply _posts/2019-11-19-s2o-how-it-was.markdown]: From a quick search; a single 'l' is the English spelling; double 'l' is the American. Leaving as is.

File: _posts/2019-11-19-s2o-how-it-was.markdown (hunk 3 above)
Comment (mechanicalpete) [reply _posts/2019-11-19-s2o-how-it-was.markdown]: Removed them all

File: _posts/2019-11-19-s2o-how-it-was.markdown [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+## Curtain Close
+
+![Pete blinking face with banner]({{ site.url }}/img/inception-pipelines/s2o-how-it-was/banner-blinking.jpg)
+
+In reflection, how was it presenting to 300 people? It was a blast!! My personal philosophy in life is that if something seems scary to you then you should just do it! And I'm very glad I did.
+
+Right before I wrap up this post, I want to thank the Latency organising committee for selecting me and giving me the opportunity to present. I believe the speaker videos will be up on the [Latency website](https://www.latencyconf.io/#/) soonish if you missed me at Latency, or just want to rewatch me in glorious action. And if you want to see the code, it's up on [GitHub here](https://github.com/mechanicalpete/serverless-to-observerless).
+
+As always, reach out if you'd like and coffee and a chat!
```
Comment (Hamo69) [inline _posts/2019-11-19-s2o-how-it-was.markdown]: > As always, reach out if you'd like and coffee and a chat!
to
> As always, reach out if you'd like a coffee and a chat!

Comment (mechanicalpete) [reply _posts/2019-11-19-s2o-how-it-was.markdown]: Ooh, good spotting, fixed.

**Pull request merged**